Resources/migration.yml

```diff
@@ -9,3 +9,25 @@
 
 chem_dispenser: ChemDispenser
 
+# brigmedic stuff. Remove this sometime soon.
+SpawnPointBrigmedic: null
+ClothingHeadsetBrigmedic: null
```

Comment:
I'm adding drozd atm and I think it would be beneficial to date these, I just used # 2023-05-04.

Reply:
Or even better, split it into multiple files prefixed with their date. Then there won't be merge conflicts.